code/modules/cargo/exports/large_objects.dm

```diff
@@ -148,5 +148,5 @@
 	worth += gases[/datum/gas/hypernoblium][MOLES]*1000
 	worth += gases[/datum/gas/miasma][MOLES]*40
 	worth += gases[/datum/gas/tritium][MOLES]*5
-	worth += gases[/datum/gas/pluoxium][MOLES]*50
+	worth += gases[/datum/gas/pluoxium][MOLES]*1
```

Comment:
```suggestion
	worth += gases[/datum/gas/pluoxium][MOLES]*5
```

you can adjust the scarcity later if it's still annoying.

Reply:
Ehhhh if you think so.